core/server/master/src/main/java/alluxio/master/file/meta/InodeLockList.java

```diff
@@ -167,7 +169,14 @@ public synchronized void lockWriteAndCheckNameAndParent(Inode<?> inode, Inode pa
    * @return the list of inodes locked in this lock list, in order of when the inodes were locked
```

Comment:
This should mention that this is a new/copy list. Also, please add a TODO to change this API to not return a copy.

Reply:
done